orderbook/usecase/orderbook_usecase.go

```diff
@@ -137,32 +137,95 @@ func (o *OrderbookUseCaseImpl) ProcessPool(ctx context.Context, pool sqsdomain.P
 	return nil
 }
 
+var (
+	// fetchActiveOrdersEvery is a duration in which orders are pushed to the client periodically
+	// This is an arbitrary number selected to avoid spamming the client
+	fetchActiveOrdersDuration = 5 * time.Second
+
+	// getActiveOrdersStreamChanLen is the length of the channel for active orders stream
+	// length is arbitrary number selected to avoid blocking
+	getActiveOrdersStreamChanLen = 50
+)
+
+// GetActiveOrdersStream implements mvc.OrderBookUsecase.
+func (o *OrderbookUseCaseImpl) GetActiveOrdersStream(ctx context.Context, address string) <-chan orderbookdomain.OrderbookResult {
+	// Result channel
+	c := make(chan orderbookdomain.OrderbookResult, getActiveOrdersStreamChanLen)
+
+	// Function to fetch orders
+	fetchOrders := func(ctx context.Context) {
```

Comment:
Is there any way we could reuse this in `GetActiveOrders` to reduce code duplication?

Reply:
I think atm code is quite opimised it for duplication: we're reusing common data structures; I think further optmisations would add not significant improvements vs time spent finding right abstractions due smalll differences between `fetchOrders` and `GetActiveOrders`. For example in `fetchOrders` we skip empty results, have additional logging specifc for this method.